arcgis-ios-sdk-samples/Features/Filter by definition expression or display filter/DefinitionExpressionDisplayFilterViewController.swift

```diff
@@ -0,0 +1,108 @@
+// Copyright 2016 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class DefinitionExpressionDisplayFilterViewController: UIViewController {
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            // Assign the map to the map view's map.
+            mapView.map = makeMap()
+        }
+    }
+    
+    /// The URL to the feature service, tracking incidents in San Francisco.
+    static let featureServiceURL = URL(string: "https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/rest/services/SF_311_Incidents/FeatureServer/0")
+    /// The feature layer made with the feature service URL.
+    let featureLayer = AGSFeatureLayer(featureTable: AGSServiceFeatureTable(url: featureServiceURL!))
+    /// The display filter definition to apply to the feature layer.
+    var displayFilterDefinition: AGSDisplayFilterDefinition?
+    /// The definition expression to apply to the feature layer.
+    var definitionExpression: String?
```

Comment:
Since `AGSFeatureLayer.definitionExpression` is non-nullable and resetting this value elsewhere simply sets it to an empty string, why not just set it to an empty string here.
```suggestion
    var definitionExpression: String = ""
```